### PR TITLE
minor change in the content

### PR DESCRIPTION
**Pull request by joinimran**
There was a minor mistake that was changing the whole meaning of the information. As policy tips are required so made some adjustments. 

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1219

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

@mypil FYI Please.

File: microsoft-365/compliance/use-notifications-and-policy-tips.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ You can also use the following tokens to help customize the email notification.
   
 ## Options for configuring policy tips
 
-For each rule in a DLP policy, you can configure policy tips to:
+For each rule in a DLP policy, you need configure policy tips to:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-notifications-and-policy-tips.md]:
```suggestion
For each rule in a DLP policy, you need to configure policy tips to:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #1219.

Thank you.

cc: @kenwith

**Comment by chrfox** [comment]
A change is needed, but the change made in this PR doesn't address the issue.  Closing this and making the change in a different PR.  The issue to be addressed is that when user notifications are enabled, both policy tips and email notifications are enabled simultaneously.  Email notifications cannot be enabled without policy tips being enabled and policy tips cannot be enabled without email notifications.

**Pull request closed without merging**